src/parser-babylon.js

```diff
@@ -2,7 +2,8 @@
 
 const createError = require("./parser-create-error");
 
-function parse(text) {
+function parse(text, options) {
```

Comment:
I wanted to leave room for other potential options, but I guess we can always change it back to an object if desired at that point.